index.ts

```diff
@@ -163,48 +193,55 @@ export function autocomplete ({
  * @param  {string} [$0.url]
  * @return {Promise}                A Promise that'll get resolved with the bulk geocode result
  */
-export function bulk ({
+export function bulk({
   addresses,
+  boundary,
   clientId,
   clientSecret,
-  boundary,
   focusPoint,
   url
 }: BaseQuery & {
-  addresses: Array<any>,
-  boundary?: Boundary,
-  focusPoint?: any
-}): Promise<BaseResponse> {
-  const geocoder = getGeocoder(clientId, clientSecret, url)
-  let addressesQuery = [...addresses]
-  const options = {}
+  addresses: Array<string>
+  boundary?: Boundary
+  focusPoint?: Point
+  text?: string
+}): Promise<{
+  features: GeoJSON.FeatureCollection
```

Comment:
This is not the correct type since `GeoJSON.FeatureCollection` consists of an object with the `features` key.

Reply:
Ah thank you! Have added a comment about this as well in 8251d96fcd55c1a47faa4ece03b7a53c7122f955